### PR TITLE
draft of miRNA section

### PR DESCRIPTION
**Pull request by bdo311**
I tried to keep this section compact. Thanks in advance for the feedback!

### CODE REVIEW

**Comment by brettbj** [comment]
@bdo311 others will be better positioned to do a full review, but could you change your tags to either [@doi:link] or [@arxiv:link] where available? There is a script to automatically process citations in this format (https://github.com/greenelab/deep-review/blob/master/CONTRIBUTING.md#markdown)

**Comment by bdo311** [comment]
@brettbj I've been following the instructions to use descriptive tags for the references. Is that something we're not doing anymore?

"For improved readability, you can (but are not required to) use a tag. For example, you can add a reference to the tag Zhou2015_deep_sea using the following syntax: [@tag:Zhou2015_deep_sea]. If you add references that use tags, make sure to add those tags and their corresponding citations to references/tags.tsv."

**Comment by brettbj** [comment]
@bdo311 my apologies, I didn't see that had been added as an option, looks good then, will leave for someone with more miRNA knowledge

File: sections/04_study.md [hunk 1]
```diff
@@ -266,8 +266,40 @@ application in this area.*
...
 ### Micro-RNA binding
 
-*miRNAs are important biologically, but have neural networks produced anything
-particularly notable in this area?*
+Prediction of microRNAs (miRNAs) in the genome as well as miRNA targets is of
+great interest, as they are critical components of gene regulatory networks and
+are often conserved across great evolutionary distance [@tag:Bracken2016_mirna
+@tag:Berezikov2011_mirna]. While many machine learning algorithms have been
+applied to solve these prediction tasks, they currently require extensive
+feature selection and optimization. For instance, the widely adopted state of
```
Comment (agitter) [inline sections/04_study.md]: I don't have any evidence to the contrary, but do you have a sense of how accepted it is that TargetScan is the state of the art?  There are other competitors, but I do tend to see TargetScan used frequently.

I'm wondering whether we need to hedge slightly and say TargetScan is an example of state of the art versus it is *the* state of the art.  If it is clearly the dominant existing method, I prefer to keep the text as-is.

**Comment by bdo311** [comment]
I was thinking about this as well @agitter -- not sure what super-accurate miRNA target prediction would provide, especially because the applications right now are more in basic biology than in therapeutics, where siRNAs and gapmers are preferred. Would love to hear thoughts from others who know the field better.

**Comment by agitter** [comment]
I don't have any specific ideas about the consequences of improved miRNA binding predictions either.  I'll merge this soon.  If someone wants to add something during our editing period they should feel welcome to do so.

**Pull request merged**